index.js

```diff
@@ -0,0 +1,68 @@
+const _ = require('lodash');
+
+const pluginDefaults = {
+  varietiesToInclude: ['view'],
+  fetchDirectives: {
+    'default-src': ['https:'],
+    'report-uri': 'http://localhost/csp_reports'
+  }
+};
+
+exports.register = (server, pluginOptions, next) => {
+  const options = _.defaults(pluginOptions, pluginDefaults);
+  const headerKey = 'Content-Security-Policy-Report-Only';
```

Comment:
Heh I originally had this in pluginDefaults, then somehow thought my way out of it.  Will put it back.